PIL/aliases.py

```diff
@@ -0,0 +1,12 @@
+from typing import Tuple
+
+# Type aliases; names subject to change
+LURD = Tuple[int, int, int, int]  # left, up(per), right, down = x0, y0, x1, y1
+XY = Tuple[int, int]
+Coord = XY
+Size = XY  # NOTE: All XY aliases will be interchangeable
+Matrix4 = Tuple[float, float, float, float]
+Matrix12 = Tuple[float, float, float, float, float, float, float, float, float, float, float, float]
+Mode = str
+Extrema = Union[Tuple[Any, Any], Tuple[Tuple[Any, Any], ...]]  # Any -> float?
```

Comment:
In the C layer, Extrema match line 11 above, where Any is either int/float, and each one has to be consistent and match the image type. So, Tuple[int, float] is invalid. Actually, due to a current limitation, multiband images are restricted to being 8 bit ints, so the actual type could be:
```
Extrema = Union[Tuple[float, float], Tuple[int,int], Tuple[Tuple[int,int],...]]
```

For pixels, text is a special case of input. I'd have to check the code to see if the single band float came back in a tuple, but I expect that it does.